Add template entity script action documentation (#28768)
home-assistant/home-assistant.github.io: source/_integrations/template.markdown

  description: The unique ID for this config block. This will be prefixed to all unique IDs of all entities in this block.
  required: false
  type: string
action:

>   description: Define an action that is executed when the trigger fires. Optional. This can be used to interact with anything via services, in particular services with [response data](/docs/scripts/service-calls/#use-templates-to-handle-response-data). [See action documentation](/docs/automation/action).

Reviewer: Should we use plural instead?
For example:
```suggestion
  description: Define actions to be executed when the trigger fires. Optional. This can be used to interact with anything via services, in particular services with [response data](/docs/scripts/service-calls/#use-templates-to-handle-response-data). [See action documentation](/docs/automation/action).
```

We should explain that variables set by the action script will be available to the templates in the entities.

>   description: Define actions to be executed when the trigger fires. Optional.  Variables set by the action script are available when evaluating entity templates. This can be used to interact with anything via services, in particular services with [response data](/docs/scripts/service-calls#use-templates-to-handle-response-data). [See action documentation](/docs/automation/action).

Reviewer 2: ```suggestion
  description: Define actions to be executed when the trigger fires. Optional. Variables set by the action script are available when evaluating entity templates. This can be used to interact with anything via services, in particular services with [response data](/docs/scripts/service-calls#use-templates-to-handle-response-data). [See action documentation](/docs/automation/action).
```
